test/acceptance/features/serviceClass.feature

```diff
@@ -0,0 +1,88 @@
+Feature: Use ServicesClass resources to manage RegisteredService resources
+
+    Background:
+        Given Primaza Cluster "primaza-main" is running
+        And   Resource "backend_crd.yaml" is installed on "primaza-main"
```

Comment:
I'd suggest to use the notation "On Primaza Cluster "primaza-main", resource "backend_crd.yaml" is installed".
Otherwise it can be hard to differentiate among primaza clusters and worker ones.

Reply:
I don't see a lot of difference between deploying a resource to a Primaza cluster and deploying a resource to a worker cluster, it's still a call to `kubectl apply`.

That said, I've realized these tests need to be written against deploying to a worker cluster, not a primaza cluster.  I'll update these tests.

Reply:
Added this as a TODO; this will be a lot easier to test once the service agent infrastructure lands.  For now, I'm just testing them in the main primaza cluster.